pages/index.vue

```diff
@@ -14,7 +14,7 @@ const AsyncComponent = () => ({
   component: new Promise((resolve, reject) => {
     setTimeout(() => {
       resolve(Main)
-    }, 1000)
+    }, 100000)
```

Comment:
Would this stay at 100 seconds or is that just for demo pusposes? Also what do you think an appropriate length would be considering 5 seconds for full animation loop?

Reply:
Totally for demo purposes, yeah :)

To be honest, I'm not sure what the length should be. If it's too short, we won't notice the animation at all. If it's too long, then people wait longer. I'll test it out.